mo_gymnasium/envs/mujoco/__init__.py

```diff
@@ -12,3 +12,9 @@
     entry_point="mo_gymnasium.envs.mujoco.hopper:MOHopperEnv",
     max_episode_steps=1000,
 )
+
+register(
+    id="mo-reacher-v4",
+    entry_point="mo_gymnasium.envs.mujoco.reacher:MOReacherEnv",
+    max_episode_steps=50,
```

Comment:
I think the pybullet version has 100 steps max

Reply:
I think it is better to use 'Reacher-V4' limit. This env is not going to be an exact reproduction of the PyBullet version anyway.